frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -1,12 +1,81 @@
-import {Component} from "@angular/core";
+import {
+  ApplicationRef,
+  Component,
+  ComponentFactoryResolver,
+  Injector,
+  OnDestroy,
+  OnInit,
+  ViewContainerRef
+} from "@angular/core";
 import {BoardCacheService} from "core-app/modules/boards/board/board-cache.service";
+import {ComponentPortal, DomPortalOutlet, PortalInjector} from "@angular/cdk/portal";
+import {BoardsMenuComponent} from "core-app/modules/boards/boards-sidebar/boards-menu.component";
+import {BoardService} from "core-app/modules/boards/board/board.service";
+import {BoardDmService} from "core-app/modules/boards/board/board-dm.service";
+import {BoardListsService} from "core-app/modules/boards/board/board-list/board-lists.service";
+import {BoardConfigurationService} from "core-app/modules/boards/board/configuration-modal/board-configuration.service";
+import {BoardActionsRegistryService} from "core-app/modules/boards/board/board-actions/board-actions-registry.service";
+import {BoardStatusActionService} from "core-app/modules/boards/board/board-actions/status/status-action.service";
+import {BoardVersionActionService} from "core-app/modules/boards/board/board-actions/version/version-action.service";
+import {QueryUpdatedService} from "core-app/modules/boards/board/query-updated/query-updated.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/66ba44eecc5f75aea45cfd4151c0c01314bf9706/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L21"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>